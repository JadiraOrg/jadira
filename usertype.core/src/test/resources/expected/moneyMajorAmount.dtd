<!ELEMENT dataset (
    MONEYMAJORAMOUNT*)>

<!ELEMENT MONEYMAJORAMOUNT EMPTY>
<!ATTLIST MONEYMAJORAMOUNT
    ID CDATA #REQUIRED
    MONEY CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

