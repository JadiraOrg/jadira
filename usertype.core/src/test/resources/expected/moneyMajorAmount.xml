<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "moneyMajorAmount.dtd">
<dataset>
  <MONEYMAJORAMOUNT ID="0" MONEY="100" NAME="test_0"/>
  <MONEYMAJORAMOUNT ID="1" MONEY="100" NAME="test_1"/>
  <MONEYMAJORAMOUNT ID="2" MONEY="0" NAME="test_2"/>
  <MONEYMAJORAMOUNT ID="3" NAME="test_3"/>
</dataset>